package.json

```diff
@@ -31,7 +31,7 @@
   "devDependencies": {
     "docsify-cli": "^4.4.2",
     "husky": "^4.3.8",
-    "jest": "^26.6.3",
+    "jest": "^26.6.0",
```

Comment:
I don't understand this change. If we need to use a specific version, remove `^`.

Reply:
we need a specific version that matches the one supported by react-scripts, but we don't want to prevent them from being upgraded by renovate, we want to be able to keep bumping both, but they need to be in sync

Reply:
renovate will try to bump them anyway. This needs to be fixed because in `npm update` will update this.

Reply:
nobody is supposed to run npm update :) the other reason I would like to keep the semver specifier is to avoid making an exception for this package alone. because having it or not doesn't make a difference for any practical purposes (we have a lock file too), I like the principle of least surprise

Reply:
My point is that zapping package-lock.json and running a fresh npm install should produce a working repository. If that's not the case, then let's fix the dependency.

Reply:
I don't see a reason why the repository should work when removing a file that's essential for the correct working of the repository (package-lock.json), but if you think that needs to be supported then yes, this dependency should be pinned. I'm fairly sure many repositories with a package-lock will stop working properly if you remove the lock file, because they'll get bumps that are not compatible. But ok, not a strong opinion about this, it will work either way

Reply:
There are plenty of times where package-lock.json becomes _unmaintainable_ and a user needs to remove it. I prefer not having tech debt going to into fresh project.

Reply:
Updated the PR to use the pinned version then.